src/dataunion/DataUnion.ts

```diff
@@ -506,13 +507,13 @@ export class DataUnion {
     /**
      * Admin: set admin fee (between 0.0 and 1.0) for the data union
      */
-    async setAdminFee(newFeeFraction: number) {
+    async setAdminFee(newFeeFraction: number, ethersOptions = {}) {
```

Comment:
Define the parameter type for `ethersOptions` (here + `transportMessage()` + `transportSignaturesForMessage()`)

If `ethers.js` doesn't provide the interface, maybe we need define it ourself:
```
export interface EthersOptions {
    gasPrice?: BigNumber|string
    gasLimit?: BigNumber|string
    value?: BigNumber|string
    nonce?: number
}
```

Reply:
agreed